app/views/finance/ecf/participant_declarations/void/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<% content_for :before_content, govuk_back_link(text: "Back", href: finance_participant_path(@participant_profile.user)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Void declaration</h1>
+
+    <%= 
+      govuk_summary_list do |summary_list|
+        summary_list.with_row do |row|
+          row.with_key { "Participant ID" }
+          row.with_value { @participant_profile.user_id }
+        end
+        
+        summary_list.with_row do |row|
+          row.with_key { "Declaration ID" }
+          row.with_value { @declaration.id }
+        end
+
+        summary_list.with_row do |row|
+          row.with_key { "Declaration type" }
+          row.with_value { @declaration.declaration_type }
+        end
+
+        summary_list.with_row do |row|
+          row.with_key { "Declaration date" }
+          row.with_value { @declaration.declaration_date.to_fs(:govuk) }
+        end
+
+        summary_list.with_row do |row|
+          row.with_key { "Lead provider" }
+          row.with_value { @declaration.cpd_lead_provider.name }
```

Comment:
maybe we can include the course identifier as well?

Reply:
I'll add that in 👍